python.sh

```diff
@@ -16,11 +16,10 @@ prefer_system_check:
 ---
 #!/bin/bash -ex
 
-case $ARCHITECTURE in 
+case $ARCHITECTURE in
   slc5*) ;;
   *)
-    echo "Building our own python. If you want to avoid this, please install python >= 2.7"
```

Comment:
This is probably more accurate for python itself, however wouldn't it make more sense to move the message about the missing modules to the `python-modules.sh` recipe, rather than dropping it completely?